x-pack/plugins/ml/public/embeddables/map/layer/incompatible_layer.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiFlexGroup, EuiFlexItem, EuiIcon, EuiText } from '@elastic/eui';
+
+interface Props {
+  noDataView: boolean;
+}
+
+export const IncompatibleLayer: FC<Props> = ({ noDataView }) => {
+  return (
+    <EuiFlexGroup gutterSize="s" color="subdued" data-test-subj="mlMapLayerIncompatible">
+      <EuiFlexItem grow={false}>
+        <EuiText size="s">
+          <EuiIcon type="crossInACircleFilled" color="subdued" />
+        </EuiText>
+      </EuiFlexItem>
+      <EuiFlexItem>
+        <EuiText color="subdued" size="s">
+          {noDataView === true ? (
+            <FormattedMessage
+              id="xpack.ml.embeddables.geoJobFlyout.noDataViewError"
+              defaultMessage="There is no source Data View for this layer. It cannot be used to create an anomaly detection job"
+            />
+          ) : (
+            <FormattedMessage
+              id="xpack.ml.embeddables.geoJobFlyout.noTimeFieldError"
+              defaultMessage="This source Dataview for this layer does not contain a timefield. It cannot be used to create an anomaly detection job"
```

Comment:
Suggest editing this to `The source data view for this layer does not contain a timestamp field....`

Reply:
Updated in fff680a74dc070c51ba2b0ed0f5c9ec53e2b52f2